src/nuget.rs

```diff
@@ -68,12 +68,21 @@ pub struct NugetPackage {
 }
 
 impl NugetPackage {
-    pub fn new() -> io::Result<Self> {
+    pub fn new(package_id: &str) -> io::Result<Self> {
         let buffer = Vec::new();
         let cursor = Cursor::new(buffer);
         let mut zip = ZipWriter::new(cursor);
         zip.start_file("[Content_Types].xml", FileOptions::default())?;
         zip.write_all(Self::content_types().as_bytes())?;
+        // add props_file to the folder buildTransitive\net46
+        let props_file_name = format!("buildTransitive/net46/{}.props", package_id);
```

Comment:
This should be under `build\netstandard2.0` similarly to ML.NET package. `buildTransitive` is only understood by PackageReference, not by packages.config

Reply:
```suggestion
        let props_file_name = format!("build/netstandard2.0/{}.props", package_id);
```